### PR TITLE
sys-cluster/singularity:

### PR DESCRIPTION
**Pull request by freeh4cker**
ebuild for new singularity version 2.5.2

Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: sys-cluster/singularity/singularity-2.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit autotools eutils python-single-r1
+
+DESCRIPTION="Application containers for Linux"
+HOMEPAGE="http://singularity.lbl.gov/"
```
Comment (Zlogene) [inline sys-cluster/singularity/singularity-2.5.2.ebuild]: https

File: sys-cluster/singularity/singularity-2.5.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit autotools eutils python-single-r1
```
Comment (Zlogene) [inline sys-cluster/singularity/singularity-2.5.2.ebuild]: why eutils?

File: sys-cluster/singularity/singularity-2.5.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	econf \
+		--with-userns \
+		$(usex suid "" "--disable-suid") \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	MAKEOPTS+=" -j1"
+	default
+	prune_libtool_files
```
Comment (Zlogene) [inline sys-cluster/singularity/singularity-2.5.2.ebuild]: use find on la files and `${D}` and delete them

File: sys-cluster/singularity/singularity-2.5.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+src_configure() {
+	econf \
+		--with-userns \
+		$(usex suid "" "--disable-suid") \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	MAKEOPTS+=" -j1"
```
Comment (Zlogene) [inline sys-cluster/singularity/singularity-2.5.2.ebuild]: why force it __here__ exactly?

File: sys-cluster/singularity/singularity-2.5.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+HOMEPAGE="http://singularity.lbl.gov/"
+SRC_URI="https://github.com/${PN}ware/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="cctbx-2.0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="examples static-libs +suid"
+
+RDEPEND="
+	sys-fs/squashfs-tools:0
```
Comment (Zlogene) [inline sys-cluster/singularity/singularity-2.5.2.ebuild]: please use single indent

Comment (Zlogene) [reply sys-cluster/singularity/singularity-2.5.2.ebuild]: also please fix your commit message, it states nothing currently

**Comment by monsieurp** [comment]
@freeh4cker: ping.

**Comment by juippis** [comment]
This has been updated in the tree.

**Pull request closed without merging**